README.md

```diff
@@ -41,21 +48,21 @@ case .failure(let error):
 }
 ```
 
-At the time of publish, we support the following requirements for an issuance request:
-* GroupRequirement
-* SelfAttestedClaimRequirement
-* IdTokenRequirement
-* AccessTokenRequirement
-* VerifiedIdRequirement
-* PinRequirement
-
-We support the following requirements for a presentation request:
-* VerifiedIdRequirement
+At the time of publish, we support the following requirements on a request:
+| Requirement                  	| Description 	| Supported on Type of Request 	|
+|------------------------------	|-------------	|------------------------------	|
+| GroupRequirement             	| A verifier/issuer could request multiple requirements. A GroupRequirement contains a list of requirements to support this use case.        	| Issuance/Presentation        	|
+| VerifiedIdRequirement        	| A verifier will require one or more Verified Ids to complete a presentation flow. An issuer can also request Verified Ids.        	| Issuance/Presentation        	|
```

Comment:
Do we want to add each verified ID requested would be an individual requirement?